Layers/LayerConfigurator.gd

```diff
@@ -467,6 +458,9 @@ func load_polygon_layer(db, layer_config, geo_layers_config, extended_as: Layer.
 	if get_extension_by_key(db, "extends_as", layer_config.id) == "BuildingRenderInfo":
 		if extended_as == null:
 			return load_buildings(db, layer_config, geo_layers_config)
+	elif get_extension_by_key(db, "extends_as", layer_config.id) == "GradientPolygonRenderInfo":
```

Comment:
Shouldn't this just be the default case for rendering polygons? I think we do not have a default for rendering polygons as of now so this would be a quite good fit, would it not?

Reply:
I think this is tricky: it's true that it would be a sensible default case, but then again, there are variables needed here (at least a height and a color) which the buildings do not need, since they are given attribute names for those values. How would you resolve that?